Makefile

```diff
@@ -74,8 +75,10 @@ endif
 include $(TPARTYDIR)/mshadow/make/mshadow.mk
 include $(DMLC_CORE)/make/dmlc.mk
 
-# all tge possible warning tread
-WARNFLAGS= -Wall -Wsign-compare
+include 3rdparty/ngraph-mxnet-bridge/ngraph.mk
```

Comment:
Does NGraph master use a different version of MKLDNN that's being used by MXNet master? Is it possible to align the mkldnn version so that use_ngraph and use_mkldnn can be turned on together?

Reply:
Hi @eric-haibin-lin, nGraph and direct MKLDNN integration are using different versions of MKLDNN right now. We do have a branch that can build them together, but we thought it would be better to isolate nGraph for initial experimental use. We're working internally to align release schedules over the next couple of quarters, so we should be able to align in the next quarter or so.